app/src/androidTest/kotlin/org/wordpress/aztec/demo/Matchers.kt

```diff
@@ -35,9 +36,9 @@ object Matchers {
 
             public override fun matchesSafely(view: View): Boolean {
                 if (view is EditText) {
-                    val regex = Regex(">\\s+<")
-                    val strippedText = view.text.toString().replace(regex, "><")
-                    return strippedText.equals(expected)
+                    val expectedHtml = Format.removeSourceEditorFormatting(expected, false)
```

Comment:
This is working, but I'm a little wary of stripping the formatting like this from both the expected and actual values, in case a bug is introduced into this function in the future. How about updating the expected values (to remove any whitespace) and comparing the unchanged expected values to the stripped actual HTML? (I can do this in another PR if you prefer/agree.)

Reply:
You're right. Let's do what you suggested in a separate PR.